spec/controllers/api/v2/sections_controller_spec.rb

```diff
@@ -1,128 +1,141 @@
+        get :show, params: { id: section.position + 1 }, format: :json
 
-      expect(response.status).to eq 404
+        expect(response.status).to eq 404
+      end
     end
   end
-end
 
-RSpec.describe Api::V2::SectionsController, 'GET #index' do
-  let!(:chapter1) { create :chapter, :with_section }
-  let!(:chapter2) { create :chapter, :with_section }
-  let(:section1)  { chapter1.section }
-  let(:section2)  { chapter2.section }
-  let!(:section_note) { create :section_note, section_id: section1.id }
+  # GET /api/v2/sections
+  describe '#index' do
+    let!(:section1)  { create(:chapter, :with_section).section }
+    let!(:section2)  { create(:chapter, :with_section).section }
+    let!(:section_note) { create :section_note, section_id: section1.id }
 
-  let(:pattern) do
-    {
-      data: [
-        {
-          id: section1.id.to_s,
-          type: 'section',
-          attributes: {
-            id: section1.id,
-            position: section1.position,
-            title: section1.title,
-            numeral: section1.numeral,
-            chapter_from: section1.chapter_from,
-            chapter_to: section1.chapter_to,
+    let(:pattern) do
+      {
+        data: [
+          {
+            id: section1.id.to_s,
+            type: 'section',
+            attributes: {
+              id: section1.id,
+              position: section1.position,
+              title: section1.title,
+              numeral: section1.numeral,
+              chapter_from: section1.chapter_from,
+              chapter_to: section1.chapter_to,
+            },
           },
-        },
-        {
-          id: section2.id.to_s,
-          type: 'section',
-          attributes: {
-            id: section2.id,
-            position: section2.position,
-            title: section2.title,
-            numeral: section2.numeral,
-            chapter_from: section2.chapter_from,
-            chapter_to: section2.chapter_to,
+          {
+            id: section2.id.to_s,
+            type: 'section',
+            attributes: {
+              id: section2.id,
+              position: section2.position,
+              title: section2.title,
+              numeral: section2.numeral,
+              chapter_from: section2.chapter_from,
+              chapter_to: section2.chapter_to,
+            },
           },
-        },
-      ],
-    }
+        ],
+      }
+    end
+
+    it 'returns rendered records' do
+      get :index, format: :json
+
+      expect(response.body).to match_json_expression pattern
+    end
   end
 
-  it 'returns rendered records' do
-    get :index, format: :json
+  # '/api/v2/sections/:id/chapters'
+  describe '#chapters' do
+    let(:section) { create(:chapter, :with_section).section }
 
-    expect(response.body).to match_json_expression pattern
+    it 'returns successfully' do
+      get :chapters, params: { id: section.id }, format: 'csv'
+
+      expect(response).to be_successful
+    end
```

Comment:
I refactored this spec a bit,
this test here, is the only I added, to check that it responds to '/api/v2/sections/:id/chapters' .

Reply:
You should be fine to just use the shared examples :)